libs/core-ui/src/lib/util/StatisticsUtils.ts

```diff
@@ -98,7 +98,7 @@ const generateBinaryStats: (outcomes: number[]) => ILabeledStatistic[] = (
     {
       key: BinaryClassificationMetrics.SelectionRate,
       label: localization.Interpret.Statistics.selectionRate,
-      stat: (falseNegCount + truePosCount) / total
+      stat: (falsePosCount + truePosCount) / total
```

Comment:
So basically, all that were PREDICTED positive over total, not that were "in ground truth" positive (which would have been false negative instead of false positive then) - these are the "selected ones".  That makes sense to me.  Some of the "selected ones" were just incorrectly selected.

Reply:
Correct! It has nothing to do with ground truth.